README.md

```diff
@@ -0,0 +1,76 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+# Pravega Schema Registry Repository
+
+Pravega Schema Registry is the latest service offering from Pravega family. The registry service is designed to store and manage schemas for the unstructured data stored in Pravega streams. The service is designed to not be limited to the data stored in Pravega and can serve as a general purpose management solution for storing and evolving schemas in wide variety of streaming and non streaming use cases. 
+
+It provides RESTful interface to store and manage schemas under schema groups. Users can safely evolve their schemas within the context of the schema group based on desired schema compatibility policy configured at a group level. The service has built in support for popular serialization formats in Avro, Profobuf and JSON schemas, however, users can also store and manage schemas from any serialization system. The service allows users to specify desired compatibility policies for evolution of their schemas but these are employed only for the natively supported serialization systems. 
+
+Along with providing a storage layer for schema, the service also stores and manages additional encoding information in form of codec information. Codecs could correspond to different compression or encryption used while encoding the serialized data at rest. The service generates unique identifiers for schemas and codec information pairs that users may use to tag their data with. 
+
+Please find relevant documentation and usage samples at following links:-
+- [Pravega](https://pravega.io)
+- [Schema Registry Design Proposal](https://github.com/pravega/schema-registry/wiki/PDP-1:-Schema-Registry)
+- [Schema Registry REST API documentation](https://github.com/pravega/schema-registry/wiki/REST-documentation)
+- [Installation Guide](https://github.com/pravega/schema-registry/wiki/Installation-Guide)
+- [REST API usage examples](https://github.com/pravega/schema-registry/wiki/REST-API-Usage-Samples)
+- [Pravega Application usage examples](https://github.com/pravega/schema-registry/wiki/Sample-Usage:-Pravega-Application)
```

Comment:
https://github.com/pravega/schema-registry/wiki/Sample-Usage - Are you sure this url is right? or we are yet to document on this url - right now its blank..

Reply:
i have updated the url from Sample-usage to sample usage, pravega application. So the link in the document is correct.